Assets/Scripts/Models/Buildable/Utility.cs

```diff
@@ -529,6 +511,26 @@ public void PauseJobs()
     }
 
     /// <summary>
+    /// Deconstructs the furniture.
```

Comment:
This doesn't deconstruct the furniture, it creates a job to deconstruct it, if it is not set to be deconstructed already.